src/style.css

```
@@ -0,0 +1,87 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
```

add class to the body and use class instead of the tag.
```suggestion
.body {
```